pandas/core/series.py

```diff
@@ -2245,14 +2230,14 @@ def unstack(self, level=-1, fill_value=None):
     # ----------------------------------------------------------------------
     # function application
 
-    def map(self, arg, na_action=None):
+    def map(self, mapper, na_action=None):
```

Comment:
Could you add a release note for this in API-breaking changes.

Reply:
reverted (was an error)